util.go

```diff
@@ -50,8 +52,7 @@ func (t ConnectionRole) String() string {
 }
 
 func newSessionID() uint64 {
-	r := rand.New(rand.NewSource(time.Now().UnixNano()))
-	return uint64(r.Uint32()*2) >> 2
+	return globalMathRandomGenerator.Uint64()
```

Comment:
In RFC, session ID is recommended to be crypto random.
Adding `randutil.CryptoUint64` https://github.com/pion/randutil/pull/2

Reply:
Using crypto rand in `NewJSEPSessionDescription` should make it returning error. It requires API change.